lips/lip-3.md

```diff
@@ -8,52 +8,52 @@ created: 06/09/2020
 ---
 
 # Introduction
-Libra Upgrade Management (LUM) is a set of protocols for the Libra Networks (LN) company to reconfigure the blockchain. With respect to time-sensitive decisions and incident response, LUM provides the Libra Networks--a subsidiary of the Libra Association licensed in Switzerland to operate a payment system--with technical means to respond to critical incidents, as (i) necessitated by its license terms and the law and (ii) as determined by the Libra Network risk appetite/tolerance policy.
+Libra Upgrade Management (LUM) is a set of protocols to reconfigure the blockchain. With respect to time-sensitive decisions and incident response, LUM provides the Libra Networks--a subsidiary of the Libra Association licensed in Switzerland to operate a payment system--with technical means to respond to critical incidents, as necessitated by compliance.
 
-It should be noted that the Association appoints LN officers. Officers are accountable for their actions, and usual measures may be taken upon deviation from their expected conduct.  Hence, the ultimate responsibility and power is by distributed governance of the Association.
+It should be noted that the Association appoints Libra Network(LN) officers. Officers are accountable for their actions, and usual measures may be taken upon deviation from their expected conduct.  Hence, the ultimate responsibility and power is by distributed governance of the Association.
```

Comment:
I personally feel like this sentence is not important for this LIP and can be removed.